packages/esm-appointments-app/src/appointments/common-components/appointments-actions.component.tsx

```diff
@@ -22,13 +22,9 @@ interface AppointmentsActionsProps {
 const AppointmentsActions: React.FC<AppointmentsActionsProps> = ({ appointment }) => {
   const { t } = useTranslation();
   const { checkInButton, checkOutButton } = useConfig<ConfigObject>();
-  const { visits, mutateVisit } = useTodaysVisits();
+  const { mutateVisit } = useTodaysVisits();
   const patientUuid = appointment.patient.uuid;
   const visitDate = dayjs(appointment.startDateTime);
-  const hasActiveVisitToday = visits?.some((visit) => visit?.patient?.uuid === patientUuid && visit?.startDatetime);
-  const hasCheckedOutToday = visits?.some(
-    (visit) => visit?.patient?.uuid === patientUuid && visit?.startDatetime && visit?.stopDatetime,
-  );
```

Comment:
As we discovered when looking at this today, this change can't be made in isolation of the changes to the checkout functionality, which currently will end the active visit.  Always ending the active visit makes sense when your checkout logic is based on the presence of an active visit.  But once you change the check-out logic to be independent of the visit status, you'll need to check that the visit is not already ended and/or prompt if the user wants to also close the visit, when you issue the modal dialog and perform the operation.

Reply:
Yeah, I likely want to tweak this.